tensorprob/model.py

```diff
@@ -67,6 +70,23 @@ def __exit__(self, e_type, e, tb):
         self.graph_ctx.__exit__(e_type, e, tb)
         self.graph_ctx = None
 
+        # Normalise all log probabilities contained in _description
+        with self._model_graph.as_default():
+            for var, (logp, integral, bounds) in self._description.items():
```

Comment:
@ibab Tensor Flow fails to calculate the gradient when the log has been normalised with bounds at infinity. Replacing infinities with +/-10^250  seems to fix the problem. Can you think of any better solutions?